Bindings/Java/Matlab/Utilities/osimTableFromStruct.m

```diff
@@ -130,35 +119,21 @@
         % append the RowVectorofVec3's to the opensim table
         osimtable.appendRow(iRow-1, row);
```

Comment:
I don't see `row` defined here. Did you mean to convert `StdVectorVec3` into `row`?

Reply:
row is defined on line 118.